packages/terra-alert/src/terra-dev-site/doc/alert/Alert.1.doc.mdx

```diff
@@ -18,8 +20,20 @@ import AlertPropsTable from 'terra-alert/src/Alert?dev-site-props-table';
 
 # Terra Alert
 
-The Terra Alert component is a notification banner that can be rendered in your application when there is information that you want to bring to the user's attention.  The Alert component supports a number of built-in notification types that render with pre-defined colors and icons that help the user understand the severity and meaning of the notification. A custom notification type is also supported that allows your application to customize an alert that may not fit into the pre-defined types.
+The Terra Alert component is a notification banner that can be rendered in your application when there is information that you want to bring to the user's attention.
 
+The Alert component supports a number of built-in notification types that render with pre-defined colors and icons that help the user understand the severity and meaning of the notification. The supported alert types and recommended uses are:
+
+- `alert` - recommended to display a critical notification for current emergencies or situations of high severity that need to be addressed promptly
```

Comment:
we mention these are recommendations above; should we drop "recommended" from these descriptions then?
```suggestion
- `alert` - displays a critical notification for current emergencies or situations of high severity that need to be addressed promptly
```

Reply:
ah, for consistency, if we make this change it should be carried through to the other descriptions